src/contracts/pods/EigenPod.sol

```diff
@@ -483,7 +475,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         validatorInfo.status = VALIDATOR_STATUS.ACTIVE;
         validatorInfo.validatorIndex = validatorIndex;
         validatorInfo.mostRecentBalanceUpdateTimestamp = oracleTimestamp;
-        validatorInfo.restakedBalanceGwei = _calculateRestakedBalanceGwei(validatorEffectiveBalanceGwei);
+        validatorInfo.restakedBalanceGwei = validatorEffectiveBalanceGwei;
```

Comment:
if we're checking against `MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR` inside of the `_verifyBalanceUpdate` function, it feels like we should here as well.
Otherwise in the event of a hard-fork we will have some weird, inconsistent behavior

Reply:
my thought is that the effective balance is capped at 32ETH, and if that changes, lets say to 64ETH, that wouldn't be an issue since we'd make MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR = 64 ETH as well.

Reply:
What is the sequencing you're imagining here?
It feels like we'd end up with inconsistent behavior unless we either (a) coordinated an upgrade _exactly at_ the hard fork or (b) coordinated an upgrade prior to the hard fork, which checked against block/slot number to know when to change the behavior.

If we are anticipating needing to upgrade for a hard fork, I'd prefer it to be possible to go through the hard fork and at least have consistent smart contract behavior prior to the upgrade.
I don't think this is a huge issue, but consistency feels safer here.
I'll ask @gpsanant to weigh in here as well

Reply:
i agree with @ChaoticWalrus. i think we will need to go the (b) path in the future, lets keep the behavior consistent for now.